boards/litex/sim/README.md

```diff
@@ -10,8 +10,8 @@ differ significantly depending on the release and configuration
 options used. This board definition currently targets and has been
 tested with
 - [the LiteX SoC generator, revision
-  444a605dea](https://github.com/enjoy-digital/litex/tree/4092180662ec62cf28b9283a020f1ff7f0892c19)
-- using the included [lx_sim.py](https://github.com/enjoy-digital/litex/blob/4092180662ec62cf28b9283a020f1ff7f0892c19/litex/tools/litex_sim.py)
+  11f7416e36](https://github.com/enjoy-digital/litex/tree/11f7416e3603bf404a3c603902162cf02551e98c)
+- using the included [lx_sim.py](https://github.com/enjoy-digital/litex/blob/11f7416e3603bf404a3c603902162cf02551e98c/litex/tools/litex_sim.py)
```

Comment:
You say`lx_sim.py` but then point to `litex_sim.py`

Reply:
Well spotted. This was done deliberately at first, because on some systems I've used (Debian) this was somehow (using the official install instructions accessible as `lx_sim` in the $PATH. However, it's now accessible was both `lxsim` and `litex_sim`. I'll change it accordingly.